wiki/Rules/Song_Content_Rules/id.md

```diff
@@ -1,32 +1,12 @@
----
-outdated: true
----
-
 # Aturan konten musik
 
 Dengan banyaknya variasi musik yang tersedia, sebagian besar lagu akan baik-baik saja untuk digunakan di osu!, asalkan tidak:
 
-- Sangat eksplisit, menjurus, atau sangat konfrontatif dan berhubungan dengan konten yang mengejutkan, cabul, atau sangat seksual
-- Mengangkat, mendukung, atau mendorong penyalahgunaan narkoba, bunuh diri, pembunuhan, atau melukai diri sendiri
-- Mendukung penekanan dan perpecahan terhadap rasial
-- Mengandung suara atau arti sangat keras dan tak tertahankan atau terlalu terpotong
 - Mengandung hal yang sangat politis
+- Mendukung tekanan dan perpecahan terhadap rasial
```

Comment:
uhh `racial tension` doesn't literally mean `perpecahan terhadap rasial` orz :(

imo you could try sth like

`Bersifat diskriminatif terhadap suatu ras ataupun kalangan tertentu`

Reply:
Better wording and translation 👍